ui/src/features/layer/layer-catalog.element.ts

```diff
@@ -110,131 +145,96 @@ export class LayerCatalog extends CoreElement {
     html`${repeat(
       this.layers,
       (node) => node,
```

Comment:
This would probably be a better key, in case the layer changes:
```suggestion
      (node) => node.layer ?? node.label ?? node,
```
Layers are definitely missing a required, unique id.